database/mongodb/mongodb.go

```diff
@@ -106,12 +108,14 @@ func (m *Mongo) Open(dsn string) (database.Driver, error) {
 
 func (m *Mongo) SetVersion(version int, dirty bool) error {
 	migrationsCollection := m.db.Collection(m.config.MigrationsCollection)
-	if err := migrationsCollection.Drop(context.TODO()); err != nil {
-		return &database.Error{OrigErr: err, Err: "drop migrations collection failed"}
-	}
-	_, err := migrationsCollection.InsertOne(context.TODO(), bson.M{"version": version, "dirty": dirty})
-	if err != nil {
-		return &database.Error{OrigErr: err, Err: "save version failed"}
+	var tr = true
```

Comment:
why not use the short declaration? e.g. `tr := true`
Also, this should only be created before it's used. e.g. right before line 117